Package@swift-5.9.swift

```diff
@@ -0,0 +1,102 @@
+// swift-tools-version:5.9
```

Comment:
`m`: I think it would be good to point out in a code comment why we need this extra Package.swift file.